crates/graphql/src/schema/objects/nft.rs

```diff
@@ -537,6 +537,22 @@ impl Collection {
         &self.0
     }
 
+    pub async fn activities(
+        &self,
+        ctx: &AppContext,
+        limit: i32,
+        offset: i32,
+    ) -> FieldResult<Vec<NftActivity>> {
+        let conn = ctx.shared.db.get()?;
+        let rows =
+            queries::metadatas::collection_activities(&conn, &self.0.address, limit, offset)?;
```

Comment:
This is going to be an n+1 query but we don't currently have it speced in the UI to display multiple collection activities at the same time so advocate to keep and address if the requirements change.

Reply:
If you can do with a data loader please give it a try.

Reply:
If I create a collection_activities loader, we will have 2 loaders of same type:

```    
pub collection_activities_loader: Loader<PublicKey<Nft>, Vec<NftActivity>>,

pub nft_activities_loader: Loader<PublicKey<Nft>, Vec<NftActivity>>,
```